source/endpoints/urls.md.erb

```diff
@@ -0,0 +1,755 @@
+country_code                             | ISO3166 alpha-2 country code
+language_code                            | ISO639 language code
+search_console_url                       | Google Search Console URL this URL matches with
+backlinks_enabled                        | whether backlinks are enabled for this URL
+backlinks_status                         | backlinks status data (read only)
+backlinks_count                          | total number of backlinks
+google_analytics_updated_at              | last time GA data was updated
+backlinks_fetching_enabled               | whether this URL has recurring backlinks fetching enabled
+include_local_pack_in_main_position      | whether to include local pack rankings in the main one (boolean, default: true)
+include_places_image_in_main_position    | whether to include image carousel rankings in the main one (boolean, default: true)
+include_featured_snippet_in_main_position| whether to include feature snippets rankings in the main one (boolean, default: true)
+include_knowledge_panel_in_main_position | whether to include knowledge panel rankings in the main one (boolean, default: true)
+crawling_session_path_ignore_pattern     | Ignore path patterns for site audits
+crawling_session_url_params_enabled      | whether to include query parameters in site audits (boolean, default: true)
+crawling_session_follow_nofollow_links   | whether to follow nofollow links in site audits (boolean, default: true)
+match_subdomains                         | whether to match subdomains (boolean, default: false)
+match_nested_urls                        | whether to match subpaths in the URL (boolean, default: true)
+site_audit_interval                      | recurring site audit intervals (in seconds, accepted valud are: 604800 (1 week) and 2592000 (30 days))
+backlinks_settings.ignored_domains       | list of domain backlinks from will be ignored
+backlinks_settings.min_url_rating        | minimum URL rating for backlink domains
+next_site_audit_crawl_at                 | next time a recurring site audit will happen (read only)
+site_audit_interval                      | recurring site audit intervals (in seconds, accepted valud are: 604800 (1 week) and 2592000 (30 days))
+backlinks_settings.ignored_domains       | list of domain backlinks from will be ignored
+backlinks_settings.min_url_rating        | minimum URL rating for backlink domains
+seo_metrics                              | Moz, Majestic, and Ahref SEO metrics
+seo_metrics_changes                      | last observed changes for Moz, Majestic, and Ahref SEO metrics
+match_subdomains                         | whether to match URL subdomains
+match_nested_urls                        | whether to match URL subpaths
+
+
+## List URLs
+
+```shell
+curl 'https://api.nightwatch.io/api/v1/urls?access_token=ACCESS_TOKEN' \\
+  -H 'Content-Type: application/json'
+```
+
+> The above command returns JSON structured like this:
+
+```json
+[
+  {
+    "id": 31234,
```

Comment:
I think pasting the whole response is unnecessary. Data is already described above. I'd just include a couple of fields, and then `...`.